src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
 }
+
+.day {
+  box-sizing: border-box;
+  width: $days-width;
+  height: $days-height;
+  background-color: #eee;
```

Comment:
Direct color values like `#eee` should be stored in variables for better maintainability and to follow the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Consider using a variable for this color.